extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitter.java

```diff
@@ -175,11 +173,12 @@ public void run()
             if (e instanceof InterruptedException) {
               Thread.currentThread().interrupt();
```

Comment:
@b-slim as far as I know it will just set interrupted flag and continue executing as normal% https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html#interrupt() The *next* blocking operation may fail immediately with `InterruptedException`.